dvc/dvcfile.py

```diff
@@ -0,0 +1,152 @@
+import os
+import re
+import logging
+
+from voluptuous import MultipleInvalid
+
+from dvc import dependency as dependency, output as output
+from dvc.stages import schema
```

Comment:
let's use `stage` instead of `stages` to match what we have in other parts of the code. I.e. `dvc/remote` `dvc/dependency` etc